lib/apiservers/engine/backends/eventmonitor.go

```diff
@@ -198,19 +210,26 @@ func (m *PortlayerEventMonitor) monitor() error {
 }
 
 // publishEvent() translate a portlayer event into a Docker event if the event is for a
-// known container and publishes them to Docker event subscribers.  Currently, it only
-// looks for stopped events.
+// known container and publishes them to Docker event subscribers.
 func (p DockerEventPublisher) PublishEvent(event plevents.BaseEvent) {
 	defer trace.End(trace.Begin(""))
 
 	vc := cache.ContainerCache().GetContainer(event.Ref)
 	if vc == nil {
 		log.Errorf("Portlayer event for container %s but not found in cache", event.Ref)
 	}
+	var dockerEvent string
+
+	switch event.Event {
+	case plevents.ContainerStopped:
+		dockerEvent = containerDieEvent
+	case plevents.ContainerStarted:
+		dockerEvent = containerStartEvent
```

Comment:
I don't really care what docker's pattern is... :-)  When I updated this I had that thought...meaning move to start..anyhoo...I'll do that..